src/client/components/GlobalError.tsx

```diff
@@ -49,6 +49,7 @@ const errorP = (left = false) => css`
 
 const errorLink = css`
   ${textColour}
+  text-decoration: underline;
```

Comment:
Link has no hover state. Added underline to make it clear to the user that this is a clickable link.